docs/starting_systemd.md

```diff
@@ -0,0 +1,121 @@
+# Starting OVOS - systemd
+OVOS being a modular system has several pieces that can start individually.
+
+user systemd service files are the recomended way to start each module, but may be started in other ways also.
+
+This is the prefered method to start the OVOS modules.  If you have not used `systemd` before, there is a lot of refrence on the web with more information.  It is out of scope of this document.  The following is assuming the user `ovos` is being used.
+
+A `systemd service` file and a `systemd hook` file is required for this to work.  We will create both files for the `ovos-messagebus` service because this is used by all other modules.  The provided `system hook` files need another python package `sdnotify` to work as written.
```

Comment:
```suggestion
A `systemd service` file and a `systemd hook` file is required for this to work.  We will create both files for the `ovos-messagebus` service because this is used by all other modules.  The provided `system hook` files need another Python package (`sdnotify`) to work as written.
```